### PR TITLE
Update ru.yml

### PR DESCRIPTION
**Pull request by rondoparisiano**

### CODE REVIEW

File: _data/langs/ru.yml [hunk 1]
```diff
@@ -15,19 +15,40 @@ nav:
...
+      development:
+        header: "Активная команда разработчиков и множество контрибьюторов"
+        paragraph: "Команда разработчиков готова к сотрудничеству с членами сообщества. Любой может написать нам и предложить свои идеи."
+      adoption:
+        header: "Готовность к внедрению"
+        paragraph: "Verge предоставляет широкий выбор защищенных кошельков для нескольких операционных систем (ОС). Революционный TOR Android кошелек обеспечивает полную анонимность при использовании на мобильных устройствах."
+      opensource:
+        header: "Децентрализованность и открытый код"
+        paragraph: "Verge – децентрализованная валюта, которая построена на платформе с открытым исходным кодом. Это независимый проект и не находится под контролем государственных органов или инвесторов."
+ resources:
```
Comment (gfranko) [inline _data/langs/ru.yml]: Could you indent this two spaces?

Comment (rondoparisiano) [inline _data/langs/ru.yml]: Sorry, which spaces are you talking about?

**Comment by gfranko** [comment]
Line 47 needs to be indented two more spaces

**Comment by rondoparisiano** [comment]
Added two spaces two line 47 before resources

**Comment by gfranko** [comment]
@rondoparisiano Now you need to remove **1 space**

**Comment by gfranko** [comment]
Someone else also contributed an FAQ Russian version, so you will need to fix any conflicts

**Comment by rondoparisiano** [comment]
Removed 1 space from line 47 before resources 
It was Alex who contributed

**Comment by rondoparisiano** [comment]
Hey! Is something else needed to be changed?

**Comment by gfranko** [comment]
@rondoparisiano someone else recently made a change to the Russian lang file, so there seems to be a conflict with your changes and theirs. Could you merge your updates with the latest changes we have?

**Comment by rondoparisiano** [comment]
Resolved conflicts

**Pull request merged**